Watchman.Web/Areas/BaseApiController.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Watchman.Web.Areas
 {
-    [Authorize]
+    //[Authorize]
```

Comment:
usuń komentarz

ogólnie chodzi o to, żeby na produkcji mieć zablokowane api - do czasu kiedy będzie działający system kont użytkowników